lms/djangoapps/discussion_api/api.py

```diff
@@ -751,20 +751,28 @@ def _do_extra_actions(api_content, cc_content, request_fields, actions_form, con
                 raise ValidationError({field: ["Invalid Key"]})
 
 
-def _handle_following_field(form_value, user, cc_content):
+def _handle_following_field(form_value, user, cc_content, api_content):
     """follow/unfollow thread for the user"""
     if form_value:
         user.follow(cc_content)
+        if cc_content["type"] == "thread":
+            api_content["read"] = True
```

Comment:
@robrap Its good to make this comment part of code base.
How about instead of adding it to specific line; we add it to method doc string (as there are multiple lines which this meaning in single method)